src/app/css-variables.ts

```diff
@@ -1,56 +1,74 @@
 import { settingsSelector } from 'app/dim-api/selectors';
+import { deepEqual } from 'fast-equals';
 import { isPhonePortraitSelector } from './shell/selectors';
-import { observeStore } from './utils/redux';
+import { StoreObserver } from './store/observerMiddleware';
 
 function setCSSVariable(property: string, value: { toString: () => string }) {
   if (value) {
     document.querySelector('html')!.style.setProperty(property, value.toString());
   }
 }
 
+export function createItemSizeObserver(): StoreObserver<number> {
+  return {
+    id: 'item-size-observer',
+    getObserved: (rs) => settingsSelector(rs).itemSize,
+    sideEffect: ({ previous, current }) => {
+      if (previous !== current) {
```

Comment:
Doesn't the listener only fire if the observed value changed?

Reply:
Yup, I stripped it out of the other instances and missed this one.